x-pack/plugins/reporting/server/browsers/chromium/driver_factory/metrics.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Metrics as PuppeteerMetrics } from 'puppeteer';
+import { cpus } from 'os';
+
+declare module 'puppeteer' {
```

Comment:
Was this added because the code returns a different object than is defined by the TS type?

Reply:
In the `puppeter`' type definitions they return `object` for the `send` method which is too generic and doesn't provide any type safety. The declaration here doesn't change the original signature but adds another override for that method.